### DISCUSSION TITLE
ELI5: How do I use a note as a template?

### DISCUSSION THREAD

**Discussion by coys997** [TriliumNext/Trilium#3477] [Q&A]
I feel like an idiot. I'm reading the documentation on Templates and none of it makes sense. It's entirely unclear what I need to do. All I want, very simply, is to create a template I can I select when adding new notes to a section. Does anyone have actual step-by-step directions on how to turn something into a template? This feels insanely more complicated than it needs to be.. 

Thank you and apologies if this is just an absolute base-level question, I just straight up can't wrap my little head around this.

**Comment by coys997** [comment]
Looks like I got it and works how I wanted it to. Anyone who needs this: 

1. Create Note
2. Under the Owned Attributes for that note enter "#template"
3. That note's name should appear as a template. Sub-notes seem to come along as well when doing so, which is what I was looking for as well. 

Ultimately, was a bit easier than I expected.

**Reply by mpagels** [reply]
Thx for the info @coys997 . Was looking for the info as well. 

The wiki could be more specific on how a template is created. I did not understand, that I had to do it that way, while reading the template part on the wiki. :-( 

Is it possible to add this information via pull request? Happy to do that.
